Package.swift

```diff
@@ -0,0 +1,25 @@
+// swift-tools-version:5.0
+//
+// SwiftSpreadsheet
+//
+
+import PackageDescription
+
+let package = Package(
+    name: "SwiftSpreadsheet",
+    products: [
+        .library(
+            name: "SwiftSpreadsheet",
+            targets: ["SwiftSpreadsheet"]),
+    ],
+    targets: [
+        // Targets are the basic building blocks of a package. A target can define a module or a test suite.
+        // Targets can depend on other targets in this package, and on products in packages which this package depends on.
```

Comment:
Would be great if you could remove these comments.

Reply:
Yeah, I forgot to remove auto-generated comments in this commit 
cc19641.